_pages/welcome-to-TTS/getting-started.md

```diff
@@ -0,0 +1,255 @@
+[some basics of the Mac OS X command line](https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line).
+
+And remember, if you can't figure out how to get it running, that's probably not your fault --- poor documentation is a defect. 
+You can file an issue in the repository or contact one of the project owners or contributors and ask them to update their README and any other relevant documentation.
+
+
+### Markdown
+
+[Markdown](https://www.markdownguide.org/cheat-sheet/) is probably the most commonly used way to 
+style text at TTS. It's used in Github, in Slack, and in many other systems. 
+If you see things that mostly look like plain text but have some semi-random * and # littered through them, 
+that's probably Markdown. It was designed to mimic how one might write a text-only email.
+
+There are quite a few online Markdown editors you can use, but after just a little practice you won't need them. 
+Pro tip: You can open a DM channel with yourself in Slack and use most Markdown in messages to yourself 
+to try it out and see the results. 
+Headers don't work in Slack, but most other Markdown commands do. 
+While you're DM-ing with yourself, you can also click the plus button on the left side of the input box, 
+click "Post", and practice your Markdown in there. 
+Slack's posts support more markdown than messages do, and it renders live. Which is pretty neat.
+
+
+### Security, Compliance, and ATOs
+
+Security and compliance are, unsurprisingly, important issues for TTS. 
+They're also complex topics, and you'll find a lot of people working on them. 
+For an easy introduction into these topics, we suggest you start with 
+[this lightweight primer](https://docs.google.com/presentation/d/1vXZDhEONp-h-zmyCt9wpwscoANouVK4fg5br6Ynzlpk/edit#slide=id.g54fbd835cd_0_25), 
+especially if you haven't worked on information security before. 
+If you're looking to go into greater detail, 
+18F's [Before You Ship](https://before-you-ship.18f.gov/) guide is definitive and should answer any questions you have, 
+but it can also be a little overwhelming. 
+If you ever need a quick answer, clarification or just general Security and Compliance help, 
+head over to [#wg-security](https://gsa-tts.slack.com/messages/CG8SNAEQM) 
+or [#wg-compliance](https://gsa-tts.slack.com/messages/C2UUQ5CNP) in Slack.
+
+If you hear "ATO" or "Authority to Operate" and are wondering what that's about, 
+[Before You Ship](https://before-you-ship.18f.gov/) and the Security and Compliance 
+leads can help you out there, too.
+
+You may also be wondering about security clearances. It's not something you need to worry about now.
+If you are in 18F and staffed to a project that needs clearance, 
+your account manager and staffing rep should make sure that any clearance needs are taken into account. 
+Still, it's never a bad idea to ask if you're wondering.
+
+
+## TTS Products and Platforms
```

Comment:
Feels a little weird to me that this section doesn't include Login, but I recognize I may be coming with bias

Reply:
I can't even recall how we developed this list and have no strong opinions. I'd probably leave it to a follow-up PR though.